tools/Raven.Debug/ThreadIo/ThreadsIoCommand.cs

```diff
@@ -0,0 +1,174 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Threading;
+
+namespace Raven.Debug.ThreadIo
+{
+    internal static class ThreadsIoCommand
+    {
+        private sealed class ThreadIoSnapshot
+        {
+            public ulong Syscr;
+            public ulong Syscw;
+            public ulong ReadBytes;
+            public ulong WriteBytes;
+            public DateTime TimestampUtc;
+        }
+
+        private sealed class ThreadIoTotals
+        {
+            public long OpsTotal;
+            public double MbTotal;
+            public long ReadOpsTotal;
+            public long WriteOpsTotal;
+            public double ReadMbTotal;
+            public double WriteMbTotal;
+        }
+
+        public static int Run(int pid, int intervalMs, TextWriter output, CancellationToken token)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                return 0; // only Linux supported
+
+            var taskDir = $"/proc/{pid}/task";
+            if (Directory.Exists(taskDir) == false)
+            {
+                output.WriteLine($"{{\"error\":\"task directory not found\",\"pid\":{pid}}}");
```

Comment:
"task directory not found" isn't very clear message error. Wouldn't be better to write explicitly that "/proc/{pid}/task was not found"?